### ISSUE TITLE
Preprint reference

### ISSUE THREAD

**Issue by fred-atherden** [libero/editor#103]
Reference to a preprint e.g. an article posted to bioRxiv.

### Author
- 
### Production staff
- 

### But what if . . . ?
- An older article is open, where the 'publication-type' attribute type is set to 'pre-print'? The solution would be to support this for opening, but when/if the article is saved the the editor will transparently to the user correct 'pre-print' to 'preprint'.

### Considerations
- At eLife, the minimum information for this reference type is:
  - Authors
  - Year
  - Title
  - Preprint site name
  - DOI or URL

## XML requirements
eLife uses the attribute `publication-type="preprint"`.
```xml
<element-citation publication-type="preprint">
    <person-group person-group-type="author">
        <name>
            <surname>Murray</surname>
            <given-names>D</given-names>
        </name>
        ...
    </person-group>
    <year iso-8601-date="2019">2019</year>
    <article-title>Gender and international diversity improves equity in peer review</article-title>
    <source>bioRxiv</source>
    <pub-id pub-id-type="biorxiv">400515</pub-id>
    <pub-id pub-id-type="doi">10.1101/400515</pub-id>
</element-citation>
```

eLife allows the following children of `element-citation[@publication-type="preprint"]`:
- person-group
- year
- article-title
- source
- pub-id (can have numerous ones)
- ext-link

`person-group` can contain child `name` or `collab`.

It would make sense to mandate the following elements:
- person-group (containing at least one `name` or `collab`)
- year
- article-title
- source

Taken from: https://github.com/elifesciences/TextureRequirements/issues/77

**Comment by NickDuf** [comment]
No user stories at present to check this feature against

**Comment by fred-atherden** [comment]
XML change: `@publication-type="preprint"` -> `@publication-type="pre-print"`.

**Comment by fred-atherden** [comment]
I think [the above change](https://github.com/libero/editor/issues/103#issuecomment-545878928) was attempting to anticipate JATS4R recommendations.

However, they have just [been released for public review](https://jats4r.org/preprint-citations), and instead mandate our current capture `@publication-type="preprint"`.

@Melissa37, we OK to change it back?

**Comment by Melissa37** [comment]
@FAtherden-eLife Yes please!

**Comment by fred-atherden** [comment]
@NuclearRedeye FYI - please can we use `<element-citation publication-type="preprint">` instead of `<element-citation publication-type="pre-print">`.

**Comment by NuclearRedeye** [comment]
@FAtherden-eLife Sure, is there any need to support 'pre-print' for legacy reasons at all? As in, should I support being able to open an article where with a citation that has the 'publication-type' set to 'pre-print'. Once open, if you were to save the document, it would write out 'preprint' instead hence transparently updating the legacy version to the newer, preferred, method.

**Comment by fred-atherden** [comment]
@NuclearRedeye I think that would be good (not for eLife legacy reasons), but in order to ensure that Libero Editor conforms to, and supports, JATS4R recommendations.

**Comment by NuclearRedeye** [comment]
@FAtherden-eLife Understood, I've updated the main issue text to account for this use case. @NickDuf This could just all be done completely transparently to the user, but that might not be desirable as I figure that maybe the user might want some indication that the tool has made a correction on their behalf, or maybe the tool needs to raise it as a suggestion and let the user decide to resolve it or not? I'm not sure if you've explored this at all in your interviews or designs, and I don't think that it's a requirement for MVP but something we need to think about.

**Comment by fred-atherden** [comment]
Thanks Joel. @Melissa37 I take it you're happy with [this](https://github.com/libero/editor/issues/103#issuecomment-607746641)?

**Comment by Melissa37** [comment]
Yeah, that sounds good. I think pre-print came from the other consortium members and not sure they adapt to JATS4R as quickly as possible, although I know they do support it where they can.

**Comment by fred-atherden** [comment]
Updated in line with the latest [JATS4R recommendations](https://jats4r.org/preprint-citations) - `elocation-id` changed to `pub-id[@pub-id-type]`.

**Comment by fred-atherden** [comment]
@NickDuf, @ilya-zhuravel FYI. Slight change to what elements are allowed for preprint refs (`elocation-id` no longer needed). I've also updated [the spreadsheet](https://docs.google.com/spreadsheets/d/1pwFkpN-StJq4jZi62G1BAsnppafVEJkoyu_FPPiRFSU/edit#gid=0).

Sorry for missing this before.
